src/ui/qgsdualviewbase.ui

```diff
@@ -182,6 +159,12 @@
    <extends>QListView</extends>
    <header>attributetable/qgsfeaturelistview.h</header>
   </customwidget>
+  <customwidget>
+   <class>QgsFieldConditionalFormatWidget</class>
+   <extends>QWidget</extends>
+   <header>qgsfieldconditionalformatwidget.h</header>
```

Comment:
this gives a build error - looks like it should be attributetable/qgsfieldconditionalformatwidget.h